runtime/browser/xwalk_browser_context.cc

```diff
@@ -349,6 +315,17 @@ net::URLRequestContextGetter*
 #endif
 }
 
+net::URLRequestContextGetter* XWalkBrowserContext::CreateMediaRequestContext() {
```

Comment:
All those Context classes you mentioned also had either an empty implementation or just called GetURLRequestContext() before in GetMediaRequestContextForStoragePartition(). ProfileImpl is the only one that had a different implementation, which was kept in the new code. I maintain that it's safer to keep the code than to remove it just because you're not sure if it's necessary -- less code being touched means a smaller surface of things that can break.